backend/api/v1beta1/python_http_client/.gitlab-ci.yml

```diff
@@ -0,0 +1,33 @@
+# ref: https://docs.gitlab.com/ee/ci/README.html
```

Comment:
What is this file for?

Reply:
all of these are auto generated via: https://github.com/kubeflow/pipelines/blob/master/test/release/release.sh 

I'll omit these for now, but we will need a follow up pr to update this script.